<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>starbook</class>
 <widget class="QMainWindow" name="starbook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>STARBOOK</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../imagenes/logoLibreria.svg</normaloff>../imagenes/logoLibreria.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <widget class="QPushButton" name="btnInicio">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background-color: rgb(50, 145, 165);
}
QPushButton{
	
	background-color: rgb(255, 138, 42);
	color: rgb(255,255,255)
}
QPushButton:hover{
background-color: rgb(255, 104, 3);

}</string>
    </property>
    <property name="text">
     <string>INICIAR SESIÓN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtClave">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,10);
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Contraseña</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtUsuario">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,10);
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Usuario</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblIniciarSesion">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(85, 85, 255,10);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imagenes/logoLibreria.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblClave_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../prjEscritorJsse/Imagenes/password.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblUsuario_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../prjEscritorJsse/Imagenes/user.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-50</x>
      <y>-10</y>
      <width>811</width>
      <height>611</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imagenes/fondo.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblMensaje">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>460</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,10);
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>btnInicio</zorder>
   <zorder>txtClave</zorder>
   <zorder>txtUsuario</zorder>
   <zorder>lblIniciarSesion</zorder>
   <zorder>lblClave_2</zorder>
   <zorder>lblUsuario_2</zorder>
   <zorder>lblMensaje</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
